src/openrct2/object/BannerSceneryEntry.h

```diff
@@ -0,0 +1,29 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2023 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "../common.h"
+
+using ObjectEntryIndex = uint16_t;
```

Comment:
We should have a header for this, in case we change it then we don't have to modify as many files. Not required for this PR